Ronni-ronsbons/week-6/hw-react-giphy/src/components/ResultsList.js

```diff
@@ -0,0 +1,21 @@
+import React, { Component } from 'react';
+import Results from './Results.js';
+
+class ResultsList extends Component {
+  render() {
+    let results = this.props.info.map( (result, index) => {
```

Comment:
Very good. Your logic is spot on. I would recommend in the future making your components and variables a little more targeted towards what you are dealing with. For example, ResultsList is too vague. Something like GiphList is more targeted. If your application had more then one type of results from a call it could get very confusing. 
Something like this is a little easier to read. Gifs is an array mapped out from the gifs property passed down from the parent. For every gif inside of gifs create a Gif component. 
```js
let gifs = this.props.gifs.map( (gif, index) => {
return (
        <Gif key={index} gif={gif} />
      );
    });
```